lib/ASRouter.jsm

```diff
@@ -1039,8 +1039,9 @@ class _ASRouter {
         );
       }
     } else {
-      // If the user is not in a trailhead-compabtible locale, return the control experience and no experiment.
-      interrupt = "control";
+      // If the user is not in a trailhead-compabtible locale, return the join + supercharge (default) experience and no experiment.
```

Comment:
Just noting for others reviewing, this "locale-specific" behavior/code is turned off because `trailhead.firstrun.branches` has a default value.